src/SubscriptionBuilder.php

```diff
@@ -99,7 +99,7 @@ public function __construct($owner, $name, $plans = null)
     public function plan($plan, $quantity = 1)
     {
         $options = [
-            'plan' => $plan,
+            'price' => $plan,
```

Comment:
this does not affect how subscriptions are currently created? this seems confusing to me, is this taking in a plan id or the price of the plan?

Reply:
It's the same thing. Both accept a price or a plan id. Stripe has provided BC for this to ease the transition to their new Prices API. But Checkout requires it to be "price". All tests still succeed btw which means this works.